test/tx-builder/dydx-governance.spec.ts

```diff
@@ -53,6 +58,27 @@ describeContract('DydxGovernance', init, (ctx: TestContext) => {
         );
         expect(emptyVoters.size).to.be.eq(0);
       });
+
+      it('getUserBalancesPerEpoch', async () => {
+        const userBalancesPerEpoch = await txBuilder.merkleDistributorService.getUserBalancesPerEpoch();
+        expect(Object.keys(userBalancesPerEpoch).length).to.be.eq(3);
+        expect(Object.keys(userBalancesPerEpoch['2']).length).to.be.eq(40867);
+      });
+
+      it('getActiveUsersInEpoch 0', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(0);
+        expect(activeUsers.length).to.be.eq(32457);
+      });
+
+      it('getActiveUsersInEpoch 1', async () => {
+        const activeUsers = await txBuilder.merkleDistributorService.getActiveUsersInEpoch(1);
+        expect(activeUsers.length).to.be.eq(9979);
```

Comment:
Can you sanity check `txBuilder.merkleDistributorService.getActiveUsersInEpoch(5)` with our DB (just check the number of users with non-zero volume in last 30 days, theoretically should be slightly greater than the number of active traders in epoch 5).

Reply:
WYM our DB?